app/components/provider_interface/references_summary_message.rb

```diff
@@ -11,9 +11,9 @@ def initialize(references, full_name)
 
     def message
       if feedback_provided.blank?
-        "The candidate #{I18n.t('provider_interface.references.requested_message', count: feedback_requested.count)}. #{I18n.t('provider_interface.references.do_not_share_message', name: full_name)}"
+        "The candidate #{I18n.t('provider_interface.references.requested_message', count: feedback_requested.count)}."
```

Comment:
Can we change the `requested_message` and `received_message` translations to include the words 'The candidate'. It's weird that a bit of the text is here and a bit in the translation (I realise that you didn't do this, but while we're touching this, we might as well improve it)

Reply:
Do you prefer this?

https://github.com/DFE-Digital/apply-for-teacher-training/pull/10057/commits/ed1e4ca7f1ee26e9e96b2a4756da13717d8699b1